esmvaltool/utils/cmorizers/obs/cmorize_obs_WOA.py

```diff
@@ -31,14 +31,14 @@
 
 import iris
 
-from .utilities import (_set_global_atts, _convert_timeunits, _fix_coords,
-                        _fix_var_metadata, _read_cmor_config, _save_variable,
-                        constant_metadata)
+from esmvaltool.utils.cmorizers.obs.utilities import (
```

Comment:
Why change the relative import?

Reply:
[PEP 8](https://www.python.org/dev/peps/pep-0008/) states `Absolute imports are recommended`.

Reply:
But also
`However, explicit relative imports are an acceptable alternative to absolute imports, especially when dealing with complex package layouts where using absolute imports would be unnecessarily verbose`.
Apart from the verbosity, I think it makes sense to use relative imports inside the package, because it makes it much easier to rename/move modules, otherwise we will have to change all import statements every time we rename something (and we're planning on renaming stuff, see #1031).